### PR TITLE
Created a notCurrentStatus scope - all models without provided names

### PR DESCRIPTION
**Pull request by reado**
Created a scope to return all models that do not match the state name(s) provided

### CODE REVIEW

File: src/HasStatuses.php [hunk 1]
```diff
@@ -71,6 +71,29 @@ public function scopeCurrentStatus(Builder $builder, string $name)
             });
     }
 
+    /**
+     * @param string|array $names
+     *
+     * @return void
+     **/
+    public function scopeNotCurrentStatus(Builder $builder, ...$names)
```
Comment (freekmurze) [inline src/HasStatuses.php]: Nice! Could you rename this to `otherCurrentStatus`?

Comment (reado) [reply src/HasStatuses.php]: Hey @freekmurze! Yah I was actually thinking about this after I submitted the PR..

I can definitely rename it to `otherCurrentStatus` but would something like `withoutCurrentStatus` make more sense?

File: tests/HasStatusesTest.php [hunk 2]
```diff
@@ -180,4 +180,24 @@ public function it_can_return_a_string_when_calling_the_attribute()
 
         $this->assertEquals('waiting for a change', $this->testModel->status()->reason);
     }
+
+    /** @test */
+    public function it_can_find_all_models_that_do_not_have_a_status_with_the_given_name()
+    {
+        $model1 = TestModel::create(['name' => 'model1']);
```
Comment (freekmurze) [inline tests/HasStatusesTest.php]: Could let this test prove that a model without a status will be found by the scope as well?

Comment (reado) [reply tests/HasStatusesTest.php]: @freekmurze Cool, I added an or condition to also look for all models w/o a status

is this what you were thinking?

**Comment by freekmurze** [comment]
Thanks!

**Pull request merged**